ddtrace/_logger.py

```diff
@@ -37,7 +37,9 @@ def configure_ddtrace_logger():
 
     """
     ddtrace_logger = logging.getLogger("ddtrace")
-    ddtrace_logger.addHandler(logging.StreamHandler())
+    if asbool(os.environ.get("DD_TRACE_STREAM_HANDLER", "true")):
```

Comment:
🚲 🏠 ... should this name have `LOG` in it?

Reply:
sure!